docs/BEMAN_STANDARD.md

```diff
@@ -422,3 +422,21 @@ copyright notice following the SPDX license identifier.
 
 **[CPP.NAMESPACE]** RECOMMENDATION: Headers in `include/beman/<short_name>/` should export
 entities in the `beman::<short_name>` namespace.
+
+## Continous Integration
+
+**[CI.PLATFORM]** REQUIREMENT: All CI should be implemented using GitHub Actions or an CI platform that appripriatly interacts with the code control system.
```

Comment:
```suggestion
**[CI.PLATFORM]** REQUIREMENT: The CI pipeline must be implemented using GitHub Actions.
```

- check REQUIREMENT and "must" in this docs
- I don't think we need to have external cases for start
- We should pick between `REQUIREMENT: The CI pipeline must be implemented using GitHub Actions.` and `RECOMMANDATION: The CI pipeline should be implemented using GitHub Actions. These options are consistent with current doc wording.

````